### PR TITLE
Testnet docs phase 1

### PR DESCRIPTION
**Pull request by dzmitryhil**
Docs on how to init and run the corem chain.

- - -
This change is [Reviewable](https://reviewable.io/reviews/coreumfoundation/coreum/245)

### CODE REVIEW

**Review by dzmitryhil — COMMENTED** [review]
> *[Reviewable](https://reviewable.io/reviews/coreumfoundation/coreum/245)* status: 0 of 15 files reviewed, 1 unresolved discussion (waiting on @miladz68, @wojtek-coreum, and @ysv)

___
*[`docs/chain/create-validator-tx.md` line 1 at r1](https://reviewable.io/reviews/coreumfoundation/coreum/245#-NFSW0Cj8siTYibeYcAf:-NFSW0Cj8siTYibeYcAg:b-mqkd2r) ([raw file](https://github.com/coreumfoundation/coreum/blob/331e2eecd2f104b0280c7adc9610767851d6adfe/docs/chain/create-validator-tx.md#L1)):*
> ```Markdown
> # TODO(dhil) Discuss whether we need it now or "init-chain.md" is enough
> ```

@ysv Do we still need that doc? Similar we have in the "prepare-initial-binary" doc.

**Review by miladz68 — DISMISSED** [review]
> Reviewed 8 of 15 files at r1, 7 of 7 files at r2, all commit messages.
> *[Reviewable](https://reviewable.io/reviews/coreumfoundation/coreum/245)* status: all files reviewed, 2 unresolved discussions (waiting on @dzmitryhil, @wojtek-coreum, and @ysv)

___
*[`docs/chain/cli-env.md` line 14 at r2](https://reviewable.io/reviews/coreumfoundation/coreum/245#-NFhvwlF1_25w5qqvsnk:-NFhvwlF1_25w5qqvsnl:b-db5hu2) ([raw file](https://github.com/coreumfoundation/coreum/blob/c4a12089038fd4ad43ceb64de6461cd8a2b0498a/docs/chain/cli-env.md#L14)):*
> ```Markdown
> | **Faucet URL**         | http://localhost:8090     | https://api.devnet-1.coreum.dev |
> | **Cosmovisor version** | v1.3.0                    | v1.3.0                          |
> | **Cored version**      | already installed in znet | check the latest devnet release |
> ```

non-blocking
`already installed via crust` is more understandable.

**Review by dzmitryhil — COMMENTED** [review]
> *[Reviewable](https://reviewable.io/reviews/coreumfoundation/coreum/245)* status: all files reviewed, 1 unresolved discussion (waiting on @wojtek-coreum and @ysv)

___
*[`docs/chain/cli-env.md` line 14 at r2](https://reviewable.io/reviews/coreumfoundation/coreum/245#-NFhvwlF1_25w5qqvsnk:-NFmTRDi8noOT-JT34NV:bb6un1j) ([raw file](https://github.com/coreumfoundation/coreum/blob/c4a12089038fd4ad43ceb64de6461cd8a2b0498a/docs/chain/cli-env.md#L14)):*
<details><summary><i>Previously, miladz68 (milad) wrote…</i></summary><blockquote>

non-blocking
`already installed via crust` is more understandable. 
</blockquote></details>

Done

**Review by miladz68 — DISMISSED** [review]
> Reviewed 1 of 1 files at r3, all commit messages.
> *[Reviewable](https://reviewable.io/reviews/coreumfoundation/coreum/245)* status: all files reviewed, 1 unresolved discussion (waiting on @wojtek-coreum and @ysv)

**Review by ysv — CHANGES_REQUESTED** [review]
I probably need an explanation for each type of node we are going to run because they are confusing for me currently

> Reviewed 8 of 15 files at r1, 6 of 7 files at r2, 1 of 1 files at r3, all commit messages.
> *[Reviewable](https://reviewable.io/reviews/coreumfoundation/coreum/245#-:-NFs7OiD-ZAa0ssKqKdm:b561u91)* status: all files reviewed, 15 unresolved discussions (waiting on @dzmitryhil and @wojtek-coreum)

___
*[`.github/workflows/markdown-link.yml` line 2 at r3](https://reviewable.io/reviews/coreumfoundation/coreum/245#-NFrpFBJAt16lRq5czfu:-NFrpFBJAt16lRq5czfv:b5lwvnj) ([raw file](https://github.com/coreumfoundation/coreum/blob/040498a83aac9a712adb36245ef91adb18e7d9ae/.github/workflows/markdown-link.yml#L2)):*
> ```YAML
> name: check markdown links
> on:
> ```

as far as I recall there is an option it github CI to run on specific folder change only
Maybe we should add it here to not run it each time but only when docs change ?
___
*[`docs/chain/install-cored.md` line 5 at r3](https://reviewable.io/reviews/coreumfoundation/coreum/245#-NFrsqwq8s8OZo7jV6mb:-NFrsqwq8s8OZo7jV6mc:bnic7jw) ([raw file](https://github.com/coreumfoundation/coreum/blob/040498a83aac9a712adb36245ef91adb18e7d9ae/docs/chain/install-cored.md#L5)):*
> ```Markdown
> The doc provides the instruction on how to install the release binaries of the "cored".
> 
> 1. [Install "cored" with "cosmovisor"](#install-cored-with-cosmovisor)
> ```

I personally prefer to wrap different cli tools references into back-ticks (`). I thought it is a common practice

e.g: Install `cored` and `cosmovisor`
___
*[`docs/chain/install-cored.md` line 9 at r3](https://reviewable.io/reviews/coreumfoundation/coreum/245#-NFrsV_03HCR-4sriXSQ:-NFrsV_03HCR-4sriXSR:b-bmgqam) ([raw file](https://github.com/coreumfoundation/coreum/blob/040498a83aac9a712adb36245ef91adb18e7d9ae/docs/chain/install-cored.md#L9)):*
> ```Markdown
> 3. [Build from sources](#build-from-sources)
> 
> ## Install "cored" with "cosmovisor"
> ```

cored and cosmovisor ?
___
*[`docs/chain/install-cored.md` line 14 at r3](https://reviewable.io/reviews/coreumfoundation/coreum/245#-NFrrRg_2h5QvDcWpvwN:-NFrrRga85t5VXIZ0OP9:bcix5is) ([raw file](https://github.com/coreumfoundation/coreum/blob/040498a83aac9a712adb36245ef91adb18e7d9ae/docs/chain/install-cored.md#L14)):*
> ```Markdown
>     It allows automatically upgrading the "cored" on the "chain upgrade".
> 
> * Set up the CLI environment following the [doc](cli-env.md).
> ```

It is a bit confusing because in cli-env doc we refer to cored binary but here we don't have it installed 
___
*[`docs/chain/install-cored.md` line 25 at r3](https://reviewable.io/reviews/coreumfoundation/coreum/245#-NFrqjKh7GTWTthD-l2e:-NFrqjKh7GTWTthD-l2f:bie9i0x) ([raw file](https://github.com/coreumfoundation/coreum/blob/040498a83aac9a712adb36245ef91adb18e7d9ae/docs/chain/install-cored.md#L25)):*
> ```Markdown
>     ```
> 
> * Install the required util: "curl tar"
> ```

required utils: `curl` and `tar`
___
*[`docs/chain/install-cored.md` line 47 at r3](https://reviewable.io/reviews/coreumfoundation/coreum/245#-NFrv2Ng6a1Daj7Fi95n:-NFrv2Ng6a1Daj7Fi95o:b-563ddb) ([raw file](https://github.com/coreumfoundation/coreum/blob/040498a83aac9a712adb36245ef91adb18e7d9ae/docs/chain/install-cored.md#L47)):*
> ```Markdown
>     chmod +x $CORED_HOME/cosmovisor/genesis/bin/*
>     export PATH=$PATH:$CORED_HOME/cosmovisor/genesis/bin
>     
> ```

is it whitespaces ?

`····`
___
*[`docs/chain/install-cored.md` line 52 at r3](https://reviewable.io/reviews/coreumfoundation/coreum/245#-NFru8Pj5LZ2pxWbwZVi:-NFru8Pj5LZ2pxWbwZVj:b-h009gf) ([raw file](https://github.com/coreumfoundation/coreum/blob/040498a83aac9a712adb36245ef91adb18e7d9ae/docs/chain/install-cored.md#L52)):*
> ```Markdown
>     ```
> 
>   **Attention!** *Set those variables globally to be automatically set after the OS reboot.*
> ```

minor:
it is not actually about OS reboot but starting a new bash session
___
*[`docs/chain/node/prepare-final-binary.md` line 1 at r3](https://reviewable.io/reviews/coreumfoundation/coreum/245#-NFs0seVDuVG7KVlzUS4:-NFs0seVDuVG7KVlzUS5:b-rqkykh) ([raw file](https://github.com/coreumfoundation/coreum/blob/040498a83aac9a712adb36245ef91adb18e7d9ae/docs/chain/node/prepare-final-binary.md#L1)):*
> ```Markdown
> # Prepare final binary
> ```

I think this part of doc is overkill and we do not want to expose it to users
Editing golang code is definitely not a way for them
We will probably have to do this changes by ourselves and provide final binary only

To be discussed with the team
___
*[`docs/chain/node/prepare-initial-binary.md` line 58 at r3](https://reviewable.io/reviews/coreumfoundation/coreum/245#-NFs1TJD1eFE97Fnp_9y:-NFs1TJD1eFE97Fnp_9z:bvimrax) ([raw file](https://github.com/coreumfoundation/coreum/blob/040498a83aac9a712adb36245ef91adb18e7d9ae/docs/chain/node/prepare-initial-binary.md#L58)):*
> ```Markdown
> 
>   **Important** write this mnemonic phrase in a safe place.
>   It is the only way to recover your account if you ever forget your password.
> ```

forget password or loose access to a machine where private key is stored
___
*[`docs/chain/node/run-genesis-validator.md` line 41 at r3](https://reviewable.io/reviews/coreumfoundation/coreum/245#-NFs2pB1E6KuCB3K-7fj:-NFs2pB1E6KuCB3K-7fk:b8x663y) ([raw file](https://github.com/coreumfoundation/coreum/blob/040498a83aac9a712adb36245ef91adb18e7d9ae/docs/chain/node/run-genesis-validator.md#L41)):*
> ```Markdown
>   ```
> 
>   **Attention!** *Be sure that node will be automatically started after the OS reboot. Add it as an OS "service" or schedule the start using the tools you prefer.*
> ```

or schedule to start



_Code quote:_
```Markdown
or schedule the start
```
___
*[`docs/chain/node/run-seed.md` line 23 at r3](https://reviewable.io/reviews/coreumfoundation/coreum/245#-NFs3KpM0UPZI1mmYBc8:-NFs3KpM0UPZI1mmYBc9:b-gji9g8) ([raw file](https://github.com/coreumfoundation/coreum/blob/040498a83aac9a712adb36245ef91adb18e7d9ae/docs/chain/node/run-seed.md#L23)):*
> ```Markdown
>   The command will create a default node configuration
> 
> * Install the required util: "crudini"
> ```

I'm not sure we should use it just for 2 commands
___
*[`docs/chain/node/run-sentry.md` line 1 at r3](https://reviewable.io/reviews/coreumfoundation/coreum/245#-NFs64QIC5wQxq6MU-qC:-NFs64QIC5wQxq6MU-qD:bdyokiy) ([raw file](https://github.com/coreumfoundation/coreum/blob/040498a83aac9a712adb36245ef91adb18e7d9ae/docs/chain/node/run-sentry.md#L1)):*
> ```Markdown
> # Run sentry
> ```

This doc is not 100% clear for me
Needs to be discussed
___
*[`docs/chain/node/run-sentry.md` line 35 at r3](https://reviewable.io/reviews/coreumfoundation/coreum/245#-NFs5dEC3B0ZxxREcOKU:-NFs5dEC3B0ZxxREcOKV:bucbd2w) ([raw file](https://github.com/coreumfoundation/coreum/blob/040498a83aac9a712adb36245ef91adb18e7d9ae/docs/chain/node/run-sentry.md#L35)):*
> ```Markdown
>   ```
> 
>   **Attention!** *That command must be executed on the validator node.
> ```

I think it is better to put this Attention before command not after
Applies to other places also
___
*[`docs/chain/create-validator-tx.md` line 1 at r1](https://reviewable.io/reviews/coreumfoundation/coreum/245#-NFSW0Cj8siTYibeYcAf:-NFs6v8u9be4CWlktAVS:bojamyq) ([raw file](https://github.com/coreumfoundation/coreum/blob/331e2eecd2f104b0280c7adc9610767851d6adfe/docs/chain/create-validator-tx.md#L1)):*
<details><summary><i>Previously, dzmitryhil (Dzmitry Hil) wrote…</i></summary><blockquote>

@ysv Do we still need that doc? Similar we have in the "prepare-initial-binary" doc.
</blockquote></details>

seems like we can remove it
___
*[`docs/chain/wasm.md` line 27 at r3](https://reviewable.io/reviews/coreumfoundation/coreum/245#-NFrvX6J9MvASs_3gLq2:-NFrvX6J9MvASs_3gLq3:byv53pb) ([raw file](https://github.com/coreumfoundation/coreum/blob/040498a83aac9a712adb36245ef91adb18e7d9ae/docs/chain/wasm.md#L27)):*
> ```Markdown
>   The [install binaries](install-cored.md) doc describes the installation process.
> 
> * Install the required util: "jq curl"
> ```

`jq` and `curl`

**Review by dzmitryhil — COMMENTED** [review]
Sore, let's have a call and I'll explain it.

> *[Reviewable](https://reviewable.io/reviews/coreumfoundation/coreum/245#-:-NFwU20y63dr3itadEIr:b-gb9par)* status: all files reviewed, 15 unresolved discussions (waiting on @wojtek-coreum and @ysv)

___
*[`.github/workflows/markdown-link.yml` line 2 at r3](https://reviewable.io/reviews/coreumfoundation/coreum/245#-NFrpFBJAt16lRq5czfu:-NFwVMVz8RmXJ-RITOog:b1ymqys) ([raw file](https://github.com/coreumfoundation/coreum/blob/040498a83aac9a712adb36245ef91adb18e7d9ae/.github/workflows/markdown-link.yml#L2)):*
<details><summary><i>Previously, ysv (Yaroslav Savchuk) wrote…</i></summary><blockquote>

as far as I recall there is an option it github CI to run on specific folder change only
Maybe we should add it here to not run it each time but only when docs change ?
</blockquote></details>

Agree, added that flag.



_Code quote:_
```YAML
 check markdown links
```
___
*[`docs/chain/install-cored.md` line 5 at r3](https://reviewable.io/reviews/coreumfoundation/coreum/245#-NFrsqwq8s8OZo7jV6mb:-NFwVXPd4DiPyWGJUeV5:b9onllq) ([raw file](https://github.com/coreumfoundation/coreum/blob/040498a83aac9a712adb36245ef91adb18e7d9ae/docs/chain/install-cored.md#L5)):*
<details><summary><i>Previously, ysv (Yaroslav Savchuk) wrote…</i></summary><blockquote>

I personally prefer to wrap different cli tools references into back-ticks (`). I thought it is a common practice

e.g: Install `cored` and `cosmovisor`
</blockquote></details>

Agree, let'd do it that way. Updated.



_Code quote:_
```Markdown
"cored"
```
___
*[`docs/chain/install-cored.md` line 9 at r3](https://reviewable.io/reviews/coreumfoundation/coreum/245#-NFrsV_03HCR-4sriXSQ:-NFwVrpw3Soagi_EGfgi:b-896fix) ([raw file](https://github.com/coreumfoundation/coreum/blob/040498a83aac9a712adb36245ef91adb18e7d9ae/docs/chain/install-cored.md#L9)):*
<details><summary><i>Previously, ysv (Yaroslav Savchuk) wrote…</i></summary><blockquote>

cored and cosmovisor ?
</blockquote></details>

Done.



_Code quote:_
```Markdown
Install "cored" with "cosmovisor"
```
___
*[`docs/chain/install-cored.md` line 14 at r3](https://reviewable.io/reviews/coreumfoundation/coreum/245#-NFrrRg_2h5QvDcWpvwN:-NFwW7re4hNs0xUphRNx:b-eqwk43) ([raw file](https://github.com/coreumfoundation/coreum/blob/040498a83aac9a712adb36245ef91adb18e7d9ae/docs/chain/install-cored.md#L14)):*
<details><summary><i>Previously, ysv (Yaroslav Savchuk) wrote…</i></summary><blockquote>

It is a bit confusing because in cli-env doc we refer to cored binary but here we don't have it installed 
</blockquote></details>

Agree, removed that reference from the `cli-env.md`



_Code quote:_
```Markdown
  In case you want to run any type of node, it's strongly recommended to run it with the "cosmovisor".
```
___
*[`docs/chain/install-cored.md` line 25 at r3](https://reviewable.io/reviews/coreumfoundation/coreum/245#-NFrqjKh7GTWTthD-l2e:-NFwWRIIAmaYeErEbT4G:b-896fix) ([raw file](https://github.com/coreumfoundation/coreum/blob/040498a83aac9a712adb36245ef91adb18e7d9ae/docs/chain/install-cored.md#L25)):*
<details><summary><i>Previously, ysv (Yaroslav Savchuk) wrote…</i></summary><blockquote>

required utils: `curl` and `tar`
</blockquote></details>

Done.



_Code quote:_
```Markdown
Install the required util:
```
___
*[`docs/chain/install-cored.md` line 47 at r3](https://reviewable.io/reviews/coreumfoundation/coreum/245#-NFrv2Ng6a1Daj7Fi95n:-NFwXD450aZCpE78iEL4:biza4qv) ([raw file](https://github.com/coreumfoundation/coreum/blob/040498a83aac9a712adb36245ef91adb18e7d9ae/docs/chain/install-cored.md#L47)):*
<details><summary><i>Previously, ysv (Yaroslav Savchuk) wrote…</i></summary><blockquote>

is it whitespaces ?

`····`
</blockquote></details>

I removed that line. But nit shure whether you still see it or now.



_Code quote:_
```Markdown
export PATH=$PATH:$CORED_HOME/cosmovisor/genesis/bin
```
___
*[`docs/chain/install-cored.md` line 52 at r3](https://reviewable.io/reviews/coreumfoundation/coreum/245#-NFru8Pj5LZ2pxWbwZVi:-NFwY1Jj1UdBiq2rfQ22:b-6y0dfk) ([raw file](https://github.com/coreumfoundation/coreum/blob/040498a83aac9a712adb36245ef91adb18e7d9ae/docs/chain/install-cored.md#L52)):*
<details><summary><i>Previously, ysv (Yaroslav Savchuk) wrote…</i></summary><blockquote>

minor:
it is not actually about OS reboot but starting a new bash session
</blockquote></details>

Update to: `after starting a new terminal session`
___
*[`docs/chain/node/prepare-final-binary.md` line 1 at r3](https://reviewable.io/reviews/coreumfoundation/coreum/245#-NFs0seVDuVG7KVlzUS4:-NFwY9jo2GScs_T72Rbv:b53umhf) ([raw file](https://github.com/coreumfoundation/coreum/blob/040498a83aac9a712adb36245ef91adb18e7d9ae/docs/chain/node/prepare-final-binary.md#L1)):*
<details><summary><i>Previously, ysv (Yaroslav Savchuk) wrote…</i></summary><blockquote>

I think this part of doc is overkill and we do not want to expose it to users
Editing golang code is definitely not a way for them
We will probably have to do this changes by ourselves and provide final binary only

To be discussed with the team
</blockquote></details>

I agree, but that is what we should do. We need that doc, probably not here.
___
*[`docs/chain/node/prepare-initial-binary.md` line 58 at r3](https://reviewable.io/reviews/coreumfoundation/coreum/245#-NFs1TJD1eFE97Fnp_9y:-NFwYN3i2PTEVaOFvJff:be52zui) ([raw file](https://github.com/coreumfoundation/coreum/blob/040498a83aac9a712adb36245ef91adb18e7d9ae/docs/chain/node/prepare-initial-binary.md#L58)):*
<details><summary><i>Previously, ysv (Yaroslav Savchuk) wrote…</i></summary><blockquote>

forget password or loose access to a machine where private key is stored
</blockquote></details>

This is the comment that the CLI produces. I can't update it :-)



_Code quote:_
```Markdown
 It is the only way to recover your account if you ever forget your password
```
___
*[`docs/chain/node/run-genesis-validator.md` line 41 at r3](https://reviewable.io/reviews/coreumfoundation/coreum/245#-NFs2pB1E6KuCB3K-7fj:-NFwYe4fDvmns9rym6Mf:b-896fix) ([raw file](https://github.com/coreumfoundation/coreum/blob/040498a83aac9a712adb36245ef91adb18e7d9ae/docs/chain/node/run-genesis-validator.md#L41)):*
<details><summary><i>Previously, ysv (Yaroslav Savchuk) wrote…</i></summary><blockquote>

or schedule to start
</blockquote></details>

Done.
___
*[`docs/chain/node/run-seed.md` line 23 at r3](https://reviewable.io/reviews/coreumfoundation/coreum/245#-NFs3KpM0UPZI1mmYBc8:-NFwYfv-7E5I1eDRN5wS:bpynt98) ([raw file](https://github.com/coreumfoundation/coreum/blob/040498a83aac9a712adb36245ef91adb18e7d9ae/docs/chain/node/run-seed.md#L23)):*
<details><summary><i>Previously, ysv (Yaroslav Savchuk) wrote…</i></summary><blockquote>

I'm not sure we should use it just for 2 commands
</blockquote></details>

What would you recommend instead? 
___
*[`docs/chain/node/run-sentry.md` line 1 at r3](https://reviewable.io/reviews/coreumfoundation/coreum/245#-NFs64QIC5wQxq6MU-qC:-NFwYn4E47acKNMnNg9W:b6icj2b) ([raw file](https://github.com/coreumfoundation/coreum/blob/040498a83aac9a712adb36245ef91adb18e7d9ae/docs/chain/node/run-sentry.md#L1)):*
> What would you recommend instead? 
Ok, let's do it.
___
*[`docs/chain/node/run-sentry.md` line 35 at r3](https://reviewable.io/reviews/coreumfoundation/coreum/245#-NFs5dEC3B0ZxxREcOKU:-NFwZ6ZN7tG05KV0PEk2:b-896fix) ([raw file](https://github.com/coreumfoundation/coreum/blob/040498a83aac9a712adb36245ef91adb18e7d9ae/docs/chain/node/run-sentry.md#L35)):*
<details><summary><i>Previously, ysv (Yaroslav Savchuk) wrote…</i></summary><blockquote>

I think it is better to put this Attention before command not after
Applies to other places also
</blockquote></details>

Done.



_Code quote:_
```Markdown
 **Attention!** *That command must be executed o
```
___
*[`docs/chain/create-validator-tx.md` line 1 at r1](https://reviewable.io/reviews/coreumfoundation/coreum/245#-NFSW0Cj8siTYibeYcAf:-NFwVRqK7XSGtJy9iDem:b-hfpv0l) ([raw file](https://github.com/coreumfoundation/coreum/blob/331e2eecd2f104b0280c7adc9610767851d6adfe/docs/chain/create-validator-tx.md#L1)):*
<details><summary><i>Previously, ysv (Yaroslav Savchuk) wrote…</i></summary><blockquote>

seems like we can remove it
</blockquote></details>

removed



_Code quote:_
```Markdown
# TODO(dhil) Discuss whether we need it now or "init-chain.md" is enough
```
___
*[`docs/chain/wasm.md` line 27 at r3](https://reviewable.io/reviews/coreumfoundation/coreum/245#-NFrvX6J9MvASs_3gLq2:-NFwY5UkENlnhp0Dqt0c:b-896fix) ([raw file](https://github.com/coreumfoundation/coreum/blob/040498a83aac9a712adb36245ef91adb18e7d9ae/docs/chain/wasm.md#L27)):*
<details><summary><i>Previously, ysv (Yaroslav Savchuk) wrote…</i></summary><blockquote>

`jq` and `curl`
</blockquote></details>

Done.

**Review by ysv — CHANGES_REQUESTED** [review]
> *[Reviewable](https://reviewable.io/reviews/coreumfoundation/coreum/245)* status: 5 of 15 files reviewed, 16 unresolved discussions (waiting on @dzmitryhil, @miladz68, and @wojtek-coreum)

*[a discussion](https://reviewable.io/reviews/coreumfoundation/coreum/245#-NGS1syd4PRVxr0LdPEb:-NGS1syd4PRVxr0LdPEc:bmyx616) (no related file):*
Add reference to tendermint doc for each node type 

___

**Review by dzmitryhil — COMMENTED** [review]
> *[Reviewable](https://reviewable.io/reviews/coreumfoundation/coreum/245)* status: 3 of 15 files reviewed, 16 unresolved discussions (waiting on @miladz68, @wojtek-coreum, and @ysv)

*[a discussion](https://reviewable.io/reviews/coreumfoundation/coreum/245#-NGS1syd4PRVxr0LdPEb:-NGWP4eZCnyv-DoKTw4w:bfrxws2) (no related file):*
<details><summary><i>Previously, ysv (Yaroslav Savchuk) wrote…</i></summary><blockquote>

Add reference to tendermint doc for each node type 
</blockquote></details>

Added for the seed and sentry. The rest will be added on we finish with the remaining docs. 

___

**Review by ysv — CHANGES_REQUESTED** [review]
> Reviewed 4 of 10 files at r4, 5 of 5 files at r5, 1 of 1 files at r7, 2 of 2 files at r8, all commit messages.
> *[Reviewable](https://reviewable.io/reviews/coreumfoundation/coreum/245)* status: all files reviewed, 3 unresolved discussions (waiting on @dzmitryhil, @miladz68, and @wojtek-coreum)

___
*[`docs/chain/install-cored.md` line 47 at r3](https://reviewable.io/reviews/coreumfoundation/coreum/245#-NFrv2Ng6a1Daj7Fi95n:-NGWgornE66JN4mpLKOS:bicjl0v) ([raw file](https://github.com/coreumfoundation/coreum/blob/040498a83aac9a712adb36245ef91adb18e7d9ae/docs/chain/install-cored.md#L47)):*
<details><summary><i>Previously, dzmitryhil (Dzmitry Hil) wrote…</i></summary><blockquote>

I removed that line. But nit shure whether you still see it or now.
</blockquote></details>

seems like it is not there now
___
*[`docs/chain/node/run-seed.md` line 23 at r3](https://reviewable.io/reviews/coreumfoundation/coreum/245#-NFs3KpM0UPZI1mmYBc8:-NGWhK6PCA8KDHbT41Fe:b-g8bvic) ([raw file](https://github.com/coreumfoundation/coreum/blob/040498a83aac9a712adb36245ef91adb18e7d9ae/docs/chain/node/run-seed.md#L23)):*
<details><summary><i>Previously, dzmitryhil (Dzmitry Hil) wrote…</i></summary><blockquote>

What would you recommend instead? 
</blockquote></details>

probably smth like: open file and on this section change smth to another value with sample yaml/toml in doc ?

Or at least add this as an alternative instead of crudini. Because it is not some sort of commonly used popular tool and I personally would avoid installing it
___
*[`docs/chain/node/run-sentry.md` line 1 at r3](https://reviewable.io/reviews/coreumfoundation/coreum/245#-NFs64QIC5wQxq6MU-qC:-NGWhkpbEVAjY4sRU0h0:b-7lg62d) ([raw file](https://github.com/coreumfoundation/coreum/blob/040498a83aac9a712adb36245ef91adb18e7d9ae/docs/chain/node/run-sentry.md#L1)):*
<details><summary><i>Previously, dzmitryhil (Dzmitry Hil) wrote…</i></summary><blockquote>

> What would you recommend instead? 
Ok, let's do it.
</blockquote></details>

after the call it is clear )
___
*[`docs/chain/node/prepare-initial-binary.md` line 58 at r3](https://reviewable.io/reviews/coreumfoundation/coreum/245#-NFs1TJD1eFE97Fnp_9y:-NGWh5YREodR7fSMPPNg:b32wtd) ([raw file](https://github.com/coreumfoundation/coreum/blob/040498a83aac9a712adb36245ef91adb18e7d9ae/docs/chain/node/prepare-initial-binary.md#L58)):*
<details><summary><i>Previously, dzmitryhil (Dzmitry Hil) wrote…</i></summary><blockquote>

This is the comment that the CLI produces. I can't update it :-)
</blockquote></details>

ok

**Review by dzmitryhil — COMMENTED** [review]
> *[Reviewable](https://reviewable.io/reviews/coreumfoundation/coreum/245)* status: all files reviewed, 3 unresolved discussions (waiting on @miladz68, @wojtek-coreum, and @ysv)

___
*[`docs/chain/install-cored.md` line 47 at r3](https://reviewable.io/reviews/coreumfoundation/coreum/245#-NFrv2Ng6a1Daj7Fi95n:-NGXK4awEN0p8srCpf8a:bmur3t9) ([raw file](https://github.com/coreumfoundation/coreum/blob/040498a83aac9a712adb36245ef91adb18e7d9ae/docs/chain/install-cored.md#L47)):*
<details><summary><i>Previously, ysv (Yaroslav Savchuk) wrote…</i></summary><blockquote>

seems like it is not there now
</blockquote></details>

Still don't see it.  :-)
___
*[`docs/chain/node/run-seed.md` line 23 at r3](https://reviewable.io/reviews/coreumfoundation/coreum/245#-NFs3KpM0UPZI1mmYBc8:-NGX6mNNDOV_KF8F-4p9:b3ppb3a) ([raw file](https://github.com/coreumfoundation/coreum/blob/040498a83aac9a712adb36245ef91adb18e7d9ae/docs/chain/node/run-seed.md#L23)):*
<details><summary><i>Previously, ysv (Yaroslav Savchuk) wrote…</i></summary><blockquote>

probably smth like: open file and on this section change smth to another value with sample yaml/toml in doc ?

Or at least add this as an alternative instead of crudini. Because it is not some sort of commonly used popular tool and I personally would avoid installing it
</blockquote></details>

It's really hard to follow such instructions. For example, in the wasm, we use the `jq` util as well.
___
*[`docs/chain/node/run-sentry.md` line 1 at r3](https://reviewable.io/reviews/coreumfoundation/coreum/245#-NFs64QIC5wQxq6MU-qC:-NGXK31L1bvpplPjVbO6:b-896fix) ([raw file](https://github.com/coreumfoundation/coreum/blob/040498a83aac9a712adb36245ef91adb18e7d9ae/docs/chain/node/run-sentry.md#L1)):*
<details><summary><i>Previously, ysv (Yaroslav Savchuk) wrote…</i></summary><blockquote>

after the call it is clear )
</blockquote></details>

Done.

**Review by ysv — CHANGES_REQUESTED** [review]
> *[Reviewable](https://reviewable.io/reviews/coreumfoundation/coreum/245)* status: 14 of 15 files reviewed, 1 unresolved discussion (waiting on @dzmitryhil, @miladz68, and @wojtek-coreum)

___
*[`docs/chain/node/run-seed.md` line 23 at r3](https://reviewable.io/reviews/coreumfoundation/coreum/245#-NFs3KpM0UPZI1mmYBc8:-NGaO-AL01KBa2aH3Smw:b-5eo4c9) ([raw file](https://github.com/coreumfoundation/coreum/blob/040498a83aac9a712adb36245ef91adb18e7d9ae/docs/chain/node/run-seed.md#L23)):*
<details><summary><i>Previously, dzmitryhil (Dzmitry Hil) wrote…</i></summary><blockquote>

It's really hard to follow such instructions. For example, in the wasm, we use the `jq` util as well.
</blockquote></details>

lets replace `crudini` with `dasel`
https://github.com/TomWright/dasel

**Review by ysv — APPROVED** [review]
> *[Reviewable](https://reviewable.io/reviews/coreumfoundation/coreum/245)* status: 14 of 15 files reviewed, all discussions resolved (waiting on @miladz68 and @wojtek-coreum)

**Review by miladz68 — APPROVED** [review]
> Reviewed 3 of 10 files at r4, 5 of 5 files at r5, 1 of 1 files at r7, 2 of 2 files at r8, 1 of 1 files at r9, all commit messages.
> *[Reviewable](https://reviewable.io/reviews/coreumfoundation/coreum/245)* status: :shipit: complete! all files reviewed, all discussions resolved (waiting on @wojtek-coreum)
